translations/en-us.yml

```diff
@@ -95,13 +95,33 @@ general:
         preprints: Preprints
         registries: Registries
     other: Other
+    checkout: Checkout
+    metadata: Metadata
+    file_description: Description
+    type: Type
+    funding_agency: 'Funding Agency'
+    type_select: 'File Type Select'
+    example_funder: 'Funder One'
+    journal_article: 'Journal Article'
+    audio_video: 'Audio/Video'
+    dataset: 'Dataset'
+    image: 'Image'
+    model: 'Model'
+    software: 'Software'
+    book: 'Book'
+    poster: 'Poster'
+    presentation: 'Presentation'
+    tags: 'Tags'
+    science: 'science'
+    engineering: 'engineering'
+
 node_categories:
     analysis: Analysis
     communication: Communication
     data: Data
     hypothesis: Hypothesis
     instrumentation: Instrumentation
-    'methods and measures': 'Methods and measures'
+    methods_and_measures: 'Methods and measures'
```

Comment:
I don't think this should be changed. I believe we get this key from the OSF and expect it to have spaces.

Reply:
Ok, I can change this back to the original